pkg/audit/manager.go

```diff
@@ -271,20 +275,20 @@ func (ucloop *updateConstraintLoop) updateConstraintStatus(ctx context.Context,
 	unstructured.SetNestedField(instance.Object, timestamp, "status", "auditTimestamp")
 	// update constraint status violations
 	if len(violations) == 0 {
-		_, found, err := unstructured.NestedSlice(instance.Object, "status", "violations")
+		_, found, err := unstructured.NestedSlice(instance.Object, "status", "violationsByAction")
```

Comment:
I think we can leave this as `violations`. `violationsByAction` implies that `action` is a unique key per element, e.g. an approximation of a map.